bucket/insomnium.json

```
@@ -0,0 +1,33 @@
+{
+    "version": "0.1.3",
+    "description": "The Insomnia HTTP and GraphQL client with online functionality removed",
+    "homepage": "https://github.com/ArchGPT/insomnium",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/ArchGPT/insomnium/releases/download/core%400.1.3/insomnia-0.1.3-full.nupkg",
+            "hash": "sha1:32A31C4DF841145334404DA98C8758E9464ADDCE"
+        }
+    },
+    "extract_dir": "lib\\net45",
+    "shortcuts": [
+        [
+            "Insomnium.exe",
+            "Insomnium"
+        ]
+    ],
+    "checkver": {
+        "url": "https://api.github.com/repos/ArchGPT/insomnium/releases",
+        "regex": "\"core@([\\d.]+)\""
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/ArchGPT/insomnium/releases/download/core%40$version/insomnia-$version-full.nupkg"
```

```suggestion
                "url": "https://github.com/ArchGPT/insomnium/releases/download/core%40$version/insomnium-$version-full.nupkg"
```

Package name fixed in 0.2.2.

---

Thanks, I'll fix that and up the version :D